<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserManagement</class>
 <widget class="QWidget" name="UserManagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestión de Usuario</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>40</number>
   </property>
   <property name="topMargin">
    <number>30</number>
   </property>
   <property name="rightMargin">
    <number>40</number>
   </property>
   <property name="bottomMargin">
    <number>30</number>
   </property>
   <item>
    <widget class="QLabel" name="lblTitulo">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Mi Perfil</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_1">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>15</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_left">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QWidget" name="avatarContainer" native="true">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
       <widget class="QLabel" name="lblAvatarPerfil">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>200</width>
          <height>200</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid gray; border-radius: 100px; background-color: lightgray;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="Assets.qrc">:/assets/icons/avatar-default.svg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="btnCambiarAvatarPencil">
        <property name="geometry">
         <rect>
          <x>152</x>
          <y>152</y>
          <width>48</width>
          <height>48</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid gray; border-radius: 24px; background-color: white;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Assets.qrc">
          <normaloff>:/assets/icons/pencil.svg</normaloff>:/assets/icons/pencil.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_right">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="lblNickname">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>usuario123</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
    <item>
     <widget class="QLabel" name="lblNuevoNombre">
      <property name="visible">
        <bool>false</bool>
      </property>
      <property name="text">
        <string>Nuevo nombre de usuario:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtNuevoNombre">
      <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
      </property>
      <property name="visible">
        <bool>false</bool>
      </property>
      <property name="placeholderText">
        <string>Ingrese el nuevo nombre</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnGuardarNombre">
      <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
      </property>
      <property name="visible">
        <bool>false</bool>
      </property>
      <property name="text">
        <string>Guardar nombre</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblNuevoCorreo">
      <property name="visible">
        <bool>false</bool>
      </property>
      <property name="text">
        <string>Nuevo correo:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtNuevoCorreo">
      <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
      </property>
      <property name="visible">
        <bool>false</bool>
      </property>
      <property name="placeholderText">
        <string>correo@ejemplo.com</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnGuardarCorreo">
      <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
      </property>
      <property name="visible">
        <bool>false</bool>
      </property>
      <property name="text">
        <string>Guardar correo</string>
      </property>
     </widget>
    </item>
   <item>
    <widget class="QLabel" name="lblContrasenaActual">
     <property name="visible">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Contraseña Actual:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txtContrasenaActual">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="visible">
      <bool>false</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Ingrese su contraseña actual</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="visible">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="lblContrasenaNueva">
     <property name="visible">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Nueva Contraseña:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txtContrasenaNueva">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="visible">
      <bool>false</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Ingrese la nueva contraseña</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_5">
     <property name="visible">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="lblConfirmarNueva">
     <property name="visible">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Confirmar Contraseña:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txtConfirmarNueva">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="visible">
      <bool>false</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Repita la nueva contraseña</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_6">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  <item>
   <spacer name="verticalSpacer_actions">
    <property name="orientation">
    <enum>Qt::Orientation::Vertical</enum>
    </property>
    <property name="sizeHint" stdset="0">
    <size>
     <width>20</width>
     <height>10</height>
    </size>
    </property>
   </spacer>
  </item>
  <item>
   <layout class="QVBoxLayout" name="verticalLayoutActions">
    <property name="spacing">
    <number>10</number>
    </property>
    <item>
    <widget class="QPushButton" name="btnCambiarNombre">
     <property name="minimumSize">
      <size>
      <width>0</width>
      <height>35</height>
      </size>
     </property>
     <property name="text">
      <string>Cambiar nombre</string>
     </property>
    </widget>
    </item>
    <item>
    <widget class="QPushButton" name="btnCambiarCorreo">
     <property name="minimumSize">
      <size>
      <width>0</width>
      <height>35</height>
      </size>
     </property>
     <property name="text">
      <string>Cambiar correo</string>
     </property>
    </widget>
    </item>
    <item>
    <widget class="QPushButton" name="btnCambiarContrasena">
     <property name="minimumSize">
      <size>
      <width>0</width>
      <height>35</height>
      </size>
     </property>
     <property name="text">
      <string>Cambiar contraseña</string>
     </property>
    </widget>
    </item>
   </layout>
  </item>
  <item>
   <spacer name="verticalSpacer_8">
    <property name="orientation">
    <enum>Qt::Orientation::Vertical</enum>
    </property>
    <property name="sizeType">
    <enum>QSizePolicy::Policy::Expanding</enum>
    </property>
    <property name="sizeHint" stdset="0">
    <size>
     <width>20</width>
     <height>40</height>
    </size>
    </property>
   </spacer>
  </item>
  <item>
   <widget class="QPushButton" name="btnCancelar">
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>35</height>
     </size>
    </property>
    <property name="visible">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
  </item>
  <item>
   <spacer name="verticalSpacer_cancel">
    <property name="visible">
     <bool>false</bool>
    </property>
    <property name="orientation">
    <enum>Qt::Orientation::Vertical</enum>
    </property>
    <property name="sizeHint" stdset="0">
    <size>
     <width>20</width>
     <height>35</height>
    </size>
    </property>
   </spacer>
  </item>
   <item>
    <widget class="QPushButton" name="btnDesloguearse">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #ff4444; color: white;</string>
     </property>
     <property name="text">
      <string>Cerrar sesión</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Assets.qrc"/>
 </resources>
 <connections/>
</ui>
